src/crypto/SessionKeystore.h

```diff
@@ -140,6 +140,22 @@ class SessionKeystore
     virtual CHIP_ERROR DeriveSessionKeys(const HkdfKeyHandle & secretKey, const ByteSpan & salt, const ByteSpan & info,
                                          Aes128KeyHandle & i2rKey, Aes128KeyHandle & r2iKey,
                                          AttestationChallenge & attestationChallenge) = 0;
+
+    /**
+     * @brief Store key in persistent PSA storage and return a key handle for an ICD Aes key.
```

Comment:
This is a generic interface and should not mention PSA. It's worth documenting that if the input is already a persistent key handle, the function is a no-op. Otherwise, if the key references a volatile key, this gets persisted and the key handle may be updated to reference the persistent key.